KonyAccelerate/controllers/mobile/frmAgendaController/frmAgendaController.js

```diff
@@ -775,5 +775,114 @@ define({
+      var sessionTile;
+      if(index==0){
+        sessionTile=this.createSessionTile(id,"131dp");
+      }
+      else{
+        sessionTile=this.createSessionTile(id,"0dp");
+      }
+      this.view.sessionTiles.add(sessionTile);
+      this.view[id].setTitleData(sessionObj);
+      if(!kony.sdk.isNullOrUndefined(sessionObj.presenter)){
+        this.view[id].onClick=this.frmAgendaSessionSelect.bind(this);
+      }
+      
+    }
+  },
+  /**
+     *	@function createSessionTile
+     * 	@description This function is invoked to get the dynamic instance of the session tile component
+     *	@param id {String} - id of the component
+     *	@param top {String} - top of the component
+     * 	@private
+     */
+  createSessionTile:function(id,top){
+      var sessionTile = new tiles.sessionTile({
+                "autogrowMode": kony.flex.AUTOGROW_NONE,
+                "clipBounds": false,
+                "height": "152dp",
+                "id": id,
+                "isVisible": true,
+                "layoutType": kony.flex.FREE_FORM,
+                "left": "0dp",
+                "masterType": constants.MASTER_TYPE_DEFAULT,
+                "isModalContainer": false,
+                "skin": "CopyslFbox0j4c69221be4c47",
+                "top": top,
+                "width": "100%",
+                "zIndex": 1,
+                "overrides": {
+                    "sessionLocationIcon": {
+                        "src": "agendatilelocationicon.png"
+                    },
+                    "sessionTile": {
+                        "bottom": "viz.val_cleared",
+                        "height": "152dp",
+                        "isVisible": true,
+                        "left": "0dp",
+                        "top": "0dp",
+                        "width": "100%"
+                    },
+                    "sessionTimeIcon": {
+                        "src": "agendatiletimeicon.png"
+                    },
+                    "tileBGImageKony": {
+                        "isVisible": true,
+                        "src": "agendatilekony.png"
+                    }
+                }
+            }, {
+                "retainFlowHorizontalAlignment": false,
+                "overrides": {}
+            }, {
+                "overrides": {}
+            });
+    return sessionTile;
+  },
+  /**
+     *	@function setSpeakerProfile
+     * 	@description This function is used to set the speaker info to the UI Elements
+     *	@param eventObject {Obejct} -traverses the Speaker_master data and fiiter the speaker from it.
+     * 	@private
+     */
+  setSpeakerProfile:function(eventObject){
+    var id=eventObject.id;
+    var len=id.length;
+    var startIndex=eventConstants.SESSION_TILE_ID.length;
+    var sessionIndex= id.substring(startIndex,len);
+    var sessionObject=this.sessionsList[sessionIndex];
+    var speakerList=sessionObject["presenter"];
+    var speakers_master=kony.store.getItem("master_speakers");
```

Comment:
Add a null check for the object retrieved from store

Reply:
Added Null checks